src/config.js

```diff
@@ -13,7 +13,8 @@ class Config {
       label: core.getInput('label'),
       ec2InstanceId: core.getInput('ec2-instance-id'),
       iamRoleName: core.getInput('iam-role-name'),
-      keyPairName: core.getInput('key-pair-name')
+      keyPairName: core.getInput('key-pair-name'),
```

Comment:
place 2 of merge conflict: 
original wanted to add `runnerHomeDir` and remove `keyPairName` I think. I kept both.

Reply:
removed

Reply:
actually I think `runnerHomeDir` can probably go because it's used by the class